apps/game_backend/priv/repo/migrations/20240509225708_add_quest_table.exs

```diff
@@ -0,0 +1,19 @@
+defmodule GameBackend.Repo.Migrations.AddQuestTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:quests) do
+      add :description, :string
```

Comment:
If you happen to make other changes, would be good to put `null: false` in the columns here otherwise no big deal we can do that later

Reply:
I had to fix some conflicts so i'll do this